Project.toml

```diff
@@ -4,8 +4,10 @@ authors = ["Climate Modeling Alliance"]
 version = "0.5.0"
 
 [deps]
+CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
```

Comment:
Just a reminder: no packages directly depend on CLIMAParameters anymore. We've only been adding dependence in the test suites

Reply:
@charleskawczynski maybe we could discuss this when you're back next week over slack! (ClimaLSM still does depend on CLIMAParameters)

Reply:
So, we've since changed directions, and high level packages are now depending on CLIMAParameters. They should still not be needed to use the package, but only needed for helper-like functionality.